api/LNSocketAPI.ts

```diff
@@ -127,6 +123,16 @@ export default class LNSocketAPI implements AppAPI {
     return await this.call("listfunds", {});
   }
 
+  async ping(node_id: string): Promise<boolean> {
+    let response: any = await this.call("ping", { id: node_id });
+    console.log(response);
+    if(response.error == null) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
you need to put this call in a try catch, if you look how the `this.call` api works you find alone the error.

In case of error this will not work anyway because an exception is thorws